src/main/java/com/marklogic/mgmt/util/PropertySourceFactory.java

```diff
@@ -73,4 +73,13 @@ public void setCheckWithMarklogicPrefix(boolean applyMarklogicPrefix) {
 	public PropertySource getPropertySource() {
 		return propertySource;
 	}
+
+	public Integer propertyToInteger(String propertyName, String propertyValue) {
+		try {
+			return Integer.parseInt(propertyValue);
+		} catch (NumberFormatException ex) {
+			throw new NumberFormatException(format("The property %s requires a numeric value; invalid value: ‘%s'", propertyName, propertyValue));
```

Comment:
Small tweak - throw an `IllegalArgumentException` here for consistency with the exceptions that are thrown in various other places for bad values.